includes/wf_crm_webform_postprocess.inc

```diff
@@ -2162,6 +2164,10 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
         elseif ($field['type'] === 'number') {
           $sum = 0;
           foreach ((array) $val as $k => $v) {
+            //Strip any thousand separator for further calculation.
+            if ($field['data_type'] === 'Money') {
+              $v = preg_replace('/(?<=\d),(?=\d{3}\b)/', '', $v);
+            }
```

Comment:
Shouldn't we be respecting the thousands separator in crm config?

Reply:
@jitendrapurohit it's

    wf_civicrm_api('Setting', 'get', array(
      'return' => array("monetaryThousandSeparator"),
    ));

Reply:
Thanks for the comments @colemanw I think I overlooked the previous comment notification. I'll check this and update the PR.

Reply:
updated the PR with the separator mentioned in the config.

Reply:
updated the PR with the separator stored in the config.